src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -33,37 +33,37 @@ const DropdownTemplate: Story<DropdownProps> = (args: DropdownProps) => {
 export const SmallSelect = DropdownTemplate.bind({});
 
 SmallSelect.args = {
-    size: Size.Small,
+    size: DropdownSize.Small,
     menuBlocks: [
         {
             id: "block1",
             menuItems: [
                 {
                     id: "1",
                     title: "Small icon",
-                    size: Size.Small,
-                    icon: <IconAudio size={IconSize.Size16} />,
+                    size: MenuItemContentSize.Small,
+                    decorator: <IconAudio />,
                 },
                 {
                     id: "2",
                     title: "Small icon warning",
-                    size: Size.Small,
-                    icon: <IconAudio size={IconSize.Size16} />,
```

Comment:
will this break for the settings where we currently use icon?

Reply:
I suppose it will yes. I renamed the prop to `decorator` because it's also possible to add small images according to the design.